### PR TITLE
Insert many allow active models to have different column set

### PR DESCRIPTION
**Pull request by tyt2y3**
Fixes https://github.com/SeaQL/sea-orm/issues/1407

This requires a new feature in sea-query, [Value::as_null](https://docs.rs/sea-query/latest/sea_query/value/enum.Value.html#method.as_null)

### CODE REVIEW

**Comment by tyt2y3** [comment]
I believe this should have solved the problem once and for all. Since this is changing a core part of the library, we should test it more to prevent regression.

The safest way might be to have a new method `insert_many_nopanic`, but of course ideally we should replace the old method transparently.

File: src/query/insert.rs [hunk 1]
```diff
@@ -155,9 +155,67 @@ where
...
         }
+
+        if !all_values.is_empty() {
+            // filter only used column
+            self.query.columns(columns.iter().cloned().flatten());
+
+            // flag used column
+            for col in columns.iter() {
+                self.columns.push(col.is_some());
+            }
```
Comment (tyt2y3) [inline src/query/insert.rs:197]: thanks. self.columns is really for use with add(). so should we do `self.columns.clear()` first?

Comment (billy1624) [reply src/query/insert.rs:197]: Okay, reset (clear) it before altering it seems safer.

Comment (billy1624) [reply src/query/insert.rs]: Or simply replace the columns vector, isn't this more readable?

```rs
self.columns = columns.iter().map(Option::is_some).collect();
```

**Pull request merged**